backend/lib/controller.js

```diff
@@ -0,0 +1,53 @@
+class Controller {
+  constructor(facade) {
+    this.facade = facade;
+  }
+
+  create(req, res, next) {
+    this.facade.create(req.body)
+      .then(doc => res.status(201).json(doc))
+      .catch(err => next(err));
```

Comment:
Do we plan on having error handling middleware built in to address any server side errors?

Reply:
that's the plan, but idk how far i'll get